spec/models/notification_spec.rb

```diff
@@ -5,23 +5,33 @@
     it 'can belong to an event' do
       expect(subject).to belong_to(:event)
     end
-
-    it 'has many recipients'
   end
-
   describe 'validations' do
     context 'status' do
       it 'accepts a valid status' do
-        notification = build(:notification, status: 'Scheduled')
+        allow_any_instance_of(Notification).to \
+                    receive(:notification_has_at_least_one_recipient).and_return(true)
+        notification = build(:notification)
         expect(notification).to be_valid
       end
 
       it 'rejects an invalid status' do
+        allow_any_instance_of(Notification).to \
```

Comment:
Same here as what I said above. But also: I don't think this test actually tests the status. It could be invalid for a different reason...however, I think rewriting these tests can be done separate from this PR.